### PR TITLE
[Term Entry] Adding Pytorch Tensor arange

### PR DESCRIPTION
**Pull request by selenahuang017**
Description
Adding new entry on the .arange() term under PyTorch Tensors concept. It is under docs/content/pytorch/concepts/tensors/terms/arange/arange.md.

Issue Solved
Addresses issue: https://github.com/Codecademy/docs/issues/5057

Type of Change
Adding a new entry

Checklist
[x ] All writings are my own.
[x ] My entry follows the Codecademy Docs style guide.
[x ] My changes generate no new warnings.
[x ] I have performed a self-review of my own writing and code.
[x ] I have checked my entry and corrected any misspellings.
[x ] I have made corresponding changes to the documentation if needed.
[x ] I have confirmed my changes are not being pushed from my forked main branch.
[x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
[x ] I have linked any issues that are relevant to this PR in the Issues Solved section.

### CODE REVIEW

File: content/pytorch/concepts/tensors/terms/arange/arange.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  - 'Arrays'
+  - 'Data Structures'
+  - 'Deep Learning'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The **`.arange()`** method returns a tensor created from a given interval `[start, end)`, with a specified `step` size. Note: when the step size is not an integer, there may be floating point rounding errors, and therefore it is recommended to subtract a small epsilon from the end for consistency.
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/arange/arange.md:19]:
```suggestion
The **`.arange()`** method returns a tensor containing values from a given interval `[start, end)` with a specified `step` size. When the step size is not an integer, floating-point rounding errors may occur, so it is recommended to subtract a small epsilon from the `end` value for consistency.
```

File: content/pytorch/concepts/tensors/terms/arange/arange.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+Tags:
+  - 'AI'
+  - 'Arrays'
+  - 'Data Structures'
+  - 'Deep Learning'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/arange/arange.md:17]:
```suggestion
---
Title: '.arange()'
Description: 'Returns a 1-dimentional tensor with values from a specified range.'
Subjects:
  - 'AI'
  - 'Data Science'
  - 'Machine Learning'
Tags:
  - 'AI'
  - 'Arrays'
  - 'Data Structures'
  - 'Deep Learning'
CatalogContent:
  - 'intro-to-py-torch-and-neural-networks'
  - 'paths/computer-science'
---
```

Comment (mamtawardhani) [reply content/pytorch/concepts/tensors/terms/arange/arange.md:17]: The catalog content should have 2 values

File: content/pytorch/concepts/tensors/terms/arange/arange.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+The parameters are as follows:
+
+- `start`: The beginning of the range, inclusive, defaults to `0`.
+- `end`: The end of the range, exclusive. Required.
+- `step`: The step size between each tensor value. Default: `1`.
+- `out`: The output Tensor, defaults to `None`.
+- `dtype`: The datatype (`torch.dtype`) of the zeros, defaults to `None`.
+- `layout`: The layout (`torch.layout`) of the output tensor, defaults to `torch.strided`.
+- `device`: The device (`torch.device`) of the output tensor, defaults to `None`.
+- `requires_grad`: A boolean indicating whether autograd will record operations on the output tensor, defaults to `False`.
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/arange/arange.md:36]:
```suggestion
- `start`: The starting value of the range, inclusive. Defaults to `0`.
- `end`: The ending value of the range, exclusive. This parameter is required.
- `step`: The difference between each consecutive value in the range. Default value is `1`.
- `out`: A tensor in which to store the output. If `None`, a new tensor is created. Default value is `None`.
- `dtype`: The desired data type of the output tensor (`torch.dtype`). If `None`, the data type will be inferred from other input arguments. Default value is `None`.
- `layout`: The desired layout of the output tensor. Default: `torch.strided`.
- `device`: The device on which the tensor will be allocated (`torch.device`). Default: `None`.
- requires_grad: A boolean indicating whether autograd should track operations on the output tensor. Default: `False`.
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @selenahuang017 thank you for contributing to Codecademy Docs, the entry is well written! 😄 

I've suggested a few changes, could you please review and modify those at your earliest convenience? Thank you! 😃

**Review by mamtawardhani — APPROVED** [review]
Hey @selenahuang017 thank you for making the changes. The entry looks good for a second round of review 🚀

**Comment by mamtawardhani** [comment]
@cigar-galaxy82 kindly finish the second review for this ASAP, its already been 2+ weeks :)

**Comment by cigar-galaxy82** [comment]
@mamtawardhani second review done

**Review by cigar-galaxy82 — APPROVED** [review]

**Pull request merged**